platform/ios/jazzy.yml

```diff
@@ -23,53 +26,61 @@ custom_categories:
       - MGLMapViewDelegate
       - MGLStyle
       - MGLUserTrackingMode
-  - name: Annotations
+  - name: User Interaction
     children:
-      - MGLAnnotation
-      - MGLAnnotationImage
-      - MGLAnnotationView
       - MGLCalloutView
       - MGLCalloutViewDelegate
+  - name: Protocols and Abstract Classes
+    children:
+      - MGLAnnotation
+      - MGLOverlay
+      - MGLShape
       - MGLMultiPoint
-      - MGLMultiPolygon
-      - MGLMultiPolyline
+      - MGLFeature
+  - name: Primitive Shapes
+    children:
       - MGLPointAnnotation
       - MGLPointCollection
       - MGLPolygon
       - MGLPolyline
-      - MGLOverlay
-      - MGLShape
+      - MGLMultiPolygon
+      - MGLMultiPolyline
       - MGLShapeCollection
+  - name: Annotations
+    children:
+      - MGLAnnotationImage
+      - MGLAnnotationView
       - MGLUserLocation
       - MGLUserLocationAnnotationView
-  - name: Map Data
+  - name: Style Features
     children:
-      - MGLFeature
-      - MGLMultiPolygonFeature
-      - MGLMultiPolylineFeature
-      - MGLPointCollectionFeature      
       - MGLPointFeature
       - MGLPolygonFeature
       - MGLPolylineFeature
+      - MGLMultiPolygonFeature
+      - MGLMultiPolylineFeature
+      - MGLPointCollectionFeature      
       - MGLShapeCollectionFeature
+  - name: Style Sources
```

Comment:
Main reasoning here was to communicate that the source classes are associated with runtime map styling in the same way as the feature and style layer classes. Trying to avoid confusion by grouping and differentiating all of these classes from the more primitive shape classes.